babel.config.js

```diff
@@ -8,6 +8,27 @@ module.exports = (api) => {
 
 	return {
 		env: {
+			esm: {
+				plugins: [
+					[
+						'react-remove-properties',
+						{
+							properties: ['data-testid'],
+						},
+					],
+					'babel-plugin-fully-specified',
```

Comment:
This is to update any import like `from './language-helpers'`, which would resolve to `./language-helpers/index.js` or `./language-helpers.js`.

In ESM, you have to explicitly specify the file and extension, in CJS you don't have to.